.github/workflows/npm-publish.yml

```
@@ -0,0 +1,36 @@
+name: Publish CDK Lib
+
+on: 
+  release:
+    types: [published]
+
```

```suggestion

defaults:
  run:
    working-directory: cdk-lib

```

this is going to be needed for all the npm commands to operate in the subfolder